### ISSUE TITLE
Make includeSubdomains optional

### ISSUE THREAD

**Issue by Munsio** [Munsio/docker-letsencrypt-nginx-proxy-companion#1]
Currently includeSubdomains is per default true.
This causing problems when the underlying subdomain (external site) does not provide an ssl cert.

**Comment by mathuin** [comment]
Found it -- and it's already fixed at https://github.com/jwilder/nginx-proxy/commit/37323320c87a392e94f3c8e8e242afd314636fa1 !

You have a script `/app/rancher.sh` that will retrieve `nginx.tmpl` from the `master` branch of  https://github.com/jwilder/nginx-proxy/ if the file doesn't already exist locally, which is fine.

I deleted the local template and rebooted the whole thing.  The new template was retrieved, the new config was generated, and now subdomains are not a problem.

Should there be any kind of way to automatically refresh these kinds of files, or is that silly?
